src/runtime/runtime.ts

```diff
@@ -65,6 +65,39 @@ function isKeyDownDebugMode(e: KeyboardEvent) {
 	return ok
 }
 
+// Ex:
+//
+// parseDurationMs("")      // -> (error)
+// parseDurationMs("0")     // -> 0
+// parseDurationMs("100")   // -> 100
+// parseDurationMs("100ms") // -> 100
+// parseDurationMs("100s")  // -> 100,000
+// parseDurationMs("100m")  // -> 6,000,000
+//
+function parseDurationMs(durStr : any){
+	let [numStr, unit] = durStr.trim().split(/(ms|s|m)$/);
+	if(!numStr){
+		throw new Error(`parseDurationMs: expected \`ms\`, \`s\`, or \`m\`; durStr=${durStr}`)
+	}
+
+	let n = +numStr;
+  switch (unit) {
+    case 'ms':
+      // No-op
+      break;
+    case 's':
+      n *= 1000;
+      break;
+    case 'm':
+      n *= 1000 * 60;
+      break;
+    default:
+      // No-op
+      break;
+  }
+  return n;
+}
+
```

Comment:
Here's the function I want to use. And you'll see where this function is used if you scroll all the way down. Is it okay to write code here in runtime.js? or should make as a separated file?

Reply:
Yep. This works but i would put it into a separate file just because there’s a lot going on in the runtime file. That way the test file will have the same name, so it will be easy to understand what the test file and what this file do. So like `parseDuration.ts` and `parseDurationMs.test.ts`.